tests/core/request.test.ts

```diff
@@ -261,6 +293,50 @@ describe('Request properties', () => {
 
       await fetch('/', { headers: { Host: 'foo.bar:8080' } }).expect(200, { port: 8080 })
     })
+    it('should derive port from the :authority header and assign it to req.port', async () => {
+      const globalDispatcher = getGlobalDispatcher()
+      onTestFinished(() => {
+        setGlobalDispatcher(globalDispatcher)
+      })
+
+      setGlobalDispatcher(
+        new UndiciAgent({
+          connect: {
+            rejectUnauthorized: false
+          },
+          allowH2: true
+        })
+      )
+
+      const { server } = InitSecureAppAndTest(
+        (req, res) => {
+          //expect(req.get('host')).toBeUndefined()
```

Comment:
nitpick: remove the comment.

Reply:
This isn't supposed to be commented out, good catch